x/crosschain/keeper/abci.go

```diff
@@ -175,3 +200,32 @@ func CheckAndUpdateCctxGasPrice(
 
 	return gasPriceIncrease, additionalFees, nil
 }
+
+// CheckAndUpdateCCTXGasPriceBTC updates the fee rate for the given Bitcoin chain CCTX
+func CheckAndUpdateCCTXGasPriceBTC(
+	ctx sdk.Context,
+	k Keeper,
+	medianGasPrice math.Uint,
+	cctx types.CrossChainTx,
+) (math.Uint, math.Uint, error) {
+	// zetacore simply update 'GasPriorityFee', and zetaclient will use it to schedule RBF tx
+	// there is no priority fee in Bitcoin, the 'GasPriorityFee' is repurposed to store latest fee rate in sat/vB
+	cctx.GetCurrentOutboundParam().GasPriorityFee = medianGasPrice.String()
+	k.SetCrossChainTx(ctx, cctx)
+
+	return math.ZeroUint(), math.ZeroUint(), nil
+}
+
+// IsCCTXGasPriceUpdateSupported checks if the given chain supports gas price update
+func IsCCTXGasPriceUpdateSupported(chainID int64, additionalChains []zetachains.Chain) bool {
```

Comment:
Can be private method

Reply:
Yeah. The drawback is that unit tests won't be able to access it and verify 16 different chain IDs in a straightforward way.